client/src/charts/wrapped_nivo/nivo_common.js

```diff
@@ -0,0 +1,238 @@
+import './nivo_common.scss';
+import graph_text from './nivo_common.yaml';
+
+import { Fragment } from 'react';
+import MediaQuery from 'react-responsive';
+import classNames from 'classnames';
+
+import { breakpoints } from '../../core/breakpoint_defs.js';
+import { formats } from "../../core/format.js";
+import { get_formatter, infobase_colors_smart } from '../shared.js';
+import { IconTable } from '../../icons/icons.js';
+import { StatelessModal } from '../../components/index.js';
+import { create_text_maker } from '../../models/text.js';
+
+const nivo_common_text_maker = create_text_maker(graph_text);
+const create_text_maker_with_nivo_common = (additional_text) => create_text_maker([graph_text, additional_text]);
+
+const fixed_symbol_shape = ({x, y, size, fill, borderWidth, borderColor}) => (
+  <rect
+    x={x}
+    y={y}
+    transform={window.feature_detection.is_IE() ? `translate(0 -4)` : ''}
+    fill={fill}
+    strokeWidth={borderWidth}
+    stroke={borderColor}
+    width={size}
+    height={size}
+    style={{ pointerEvents: 'none' }}
+  />
+);
+const fix_legend_symbols = (legends) => legends ?
+  _.map(
+    legends,
+    legend => _.chain(legend)
+      .clone()
+      .assign({symbolShape: fixed_symbol_shape})
+      .value()
+  ) :
+  undefined;
+
+
+const DefaultTooltipColorLegend = ({tooltip_item}) => (
+  <td className="nivo-tooltip__content">
+    <div style={{height: '12px', width: '12px', backgroundColor: tooltip_item.color}} />
```

Comment:
don't know if this is long enough, but maybe expand this out

Reply:
Thought I moved the height and width to CSS, must have done that somewhere else and missed this.